samples/auth/build.gradle

```diff
@@ -86,8 +86,13 @@ dependencies {
     implementation 'com.google.android.gms:play-services-auth:20.3.0'
     implementation 'com.facebook.android:facebook-login:12.3.0'
 
-    // JWT
-    implementation 'com.nimbusds:nimbus-jose-jwt:9.0.1'
+    //Device Binding + JWT
+    implementation 'com.nimbusds:nimbus-jose-jwt:9.23'
```

Comment:
we need to add this in sample kotlin app as well for compilation issue

Reply:
kotlin sample app has the dependency.